modules/programs/git.nix

```diff
@@ -377,6 +403,19 @@ in {
         };
     })
 
+    (mkIf cfg.difftastic.enable {
```

Comment:
We should add an assertion that `cfg.difftastic.enable` and `cfg.delta.enable` are not enabled at the same time.

Reply:
Done, I figured it out